conf.py

```diff
@@ -59,7 +59,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'AT-TPC Analysis'
+project = 'AT-TPC_Analysis'
```

Comment:
Why do we need the underscore? I only ask because this string shows up as the title of the documentation on ReadTheDocs (in the top left corner): https://attpc-analysis.readthedocs.io/en/latest/